site/src/sphinx/client-circuit-breaker.rst

```diff
@@ -1,6 +1,270 @@
+            response -> response.completionFuture().handle((res, cause) -> cause == null);
+
+    // Create CircuitBreakers per host (a.com, b.com ...)
+    CircuitBreakerHttpClient.newPerHostDecorator(factory, httpStrategy);
+    CircuitBreakerRpcClient.newPerHostDecorator(factory, rpcStrategy);
+    // The names of the created CircuitBreaker: my-cb-a.com, my-cb-b.com, ...
+
+- Per Method: a single :api:`CircuitBreaker` is used for each method.
+
+.. code-block:: java
+
+    // Create CircuitBreakers per method
+    CircuitBreakerHttpClient.newPerHostDecorator(factory, httpStrategy);
+    // The names of the created CircuitBreaker: my-cb-GET, my-cb-POST, ...
+
+    CircuitBreakerRpcClient.newPerHostDecorator(factory, rpcStrategy);
+    // The names of the created CircuitBreaker: my-cb-methodA, my-cb-methodB, ...
+
+- Per Host and Method: a single :api:`CircuitBreaker` is used each method in each host.
+
+.. code-block:: java
+
+    // Create CircuitBreakers per host and method
+    CircuitBreakerHttpClient.newPerHostAndMethodDecorator(factory, httpStrategy);
+    // The names of the created CircuitBreaker: my-cb-a.com#GET,
+    // my-cb-a.com#POST, my-cb-b.com#GET, my-cb-b.com#POST, ...
+
+    CircuitBreakerRpcClient.newPerHostAndMethodDecorator(factory, rpcStrategy);
+    // The names of the created CircuitBreaker: my-cb-a.com#methodA,
+    // my-cb-a.com#methodB, my-cb-b.com#methodA, my-cb-b.com#methodB, ...
+
+If you want none of the above groupings, you can group them whatever you want using
+:api:`KeyedCircuitBreakerMapping` and :api:`KeyedCircuitBreakerMapping.KeySelector`.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.client.circuitbreaker.KeyedCircuitBreakerMapping;
+    import com.linecorp.armeria.client.circuitbreaker.KeyedCircuitBreakerMapping.KeySelector;
+
+    final KeyedCircuitBreakerMapping<String> mapping =
+            new KeyedCircuitBreakerMapping<>((ctx, req) -> ctx.path(), factory);
+    // I want to create CircuitBreakers per path!
+
+    CircuitBreakerHttpClient.newDecorator(mapping, httpStrategy);
+
+``CircuitBreakerBuilder``
+-------------------------
+
+We have used static methods in :api:`CircuitBreaker` interface to create a :api:`CircuitBreaker`.
+If you use :api:`CircuitBreakerBuilder`, you can configure the parameters which decide
+:api:`CircuitBreaker`'s behavior. Here are the parameters:
+
+- ``name``:
+
+  - The name of the :api:`CircuitBreaker`.
+
+- ``failureRateThreshold``:
+
+  - The threshold that changes :api:`CircuitBreaker`'s state to ``OPEN`` when the number of failed
+    :apiplural:`Request` divided by the number of total :apiplural:`Request` exceeds it.
+    The default value is ``0.2``.
+
+- ``minimumRequestThreshold``:
+
+  - The minimum number of :apiplural:`Request` to detect failures. The default value is ``10``.
+
+- ``trialRequestInterval``:
+
+  - The duration that a :api:`CircuitBreaker` remains in ``HALF_OPEN`` state. All requests are blocked off
```

Comment:
--

Reply:
I didn't get it.. -.-?

Reply:
Never mind. I had a problem deleting the comment, so I edited it into nothing. :sweat_smile: